android/app/src/main/java/org/fossasia/openevent/data/Microlocation.kt

```diff
@@ -0,0 +1,31 @@
+package org.fossasia.openevent.data
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import com.github.jasminb.jsonapi.IntegerIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+import io.realm.RealmObject
+import io.realm.annotations.PrimaryKey
+
+@Type("microlocation")
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+open class Microlocation(
+        @PrimaryKey
+        @Id(IntegerIdHandler::class)
+        var id: Int = 0,
+        var name: String? = null,
+        var latitude: Float = 0.toFloat(),
+        var longitude: Float = 0.toFloat(),
```

Comment:
Nullable and default to null

Reply:
Nullable and default to null is the direct Java equivalent and should be used everywhere

Reply:
If the latitude and longitude are not set, they should be null and not point to [https://en.wikipedia.org/wiki/Null_Island](Null Island). Similar logical errors will creep in if you'll assign default values to properties

Also, kotlin will throw error while parsing if JSON contains null